action.yml

```diff
@@ -69,4 +69,10 @@ runs:
         INPUT_TEMPLATE: ${{ inputs.template }}
         INPUT_UPLOAD: ${{ inputs.upload }}
         INPUT_LICENSE: ${{ inputs.license }}
+    - name: override default env
+      if: {{github.event_name}} == 'pull_request'
```

Comment:
This will cause the `run` to be skipped if it's not a PR.
I would just check for the environment variable in `bin/luarocks-tag-release-action.lua`.